### PR TITLE
Updated to version 2.1.2

### PR DESCRIPTION
**Pull request by cezary-butler**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,11 +7,11 @@
         <artifactId>oss-parent</artifactId>
         <version>7</version>
     </parent>
-    
+
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>parsleyjs</artifactId>
-    <version>2.0.7-SNAPSHOT</version>
+    <version>2.1.2</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `2.1.1-SNAPSHOT` since Maven does the actual release.

**Comment by cezary-butler** [comment]
Changed as you requested

File: pom.xml [hunk 2]
```diff
@@ -7,11 +7,11 @@
         <artifactId>oss-parent</artifactId>
         <version>7</version>
     </parent>
-    
+
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>parsleyjs</artifactId>
-    <version>2.0.7-SNAPSHOT</version>
+    <version>2.1.1-SNAPSHOT</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `2.1.2-SNAPSHOT` for a `2.1.2` release.

**Comment by jamesward** [comment]
Sorry, I messed up in my first comment with the version number.

**Comment by cezary-butler** [comment]
Is it ok now?

**Pull request merged**